{{Lund}}
<html>

<header class="design-header">
  <div class="container">
    <h1 class="display">Design</h1>
  </div>
</header>

<div class="container">
  <ul class="nav nav-pills nav-auto-stacked">
    <li role="presentation"><a href="/Team:Lund/Design">Overview</a></li>
    <li role="presentation"><a href="/Team:Lund/Design/Animation">Animation</a></li>
    <li role="presentation"><a href="/Team:Lund/Design/Theory">Theory</a></li>
    <li role="presentation" class="active"><a href="/Team:Lund/Design/Design">Project Design</a></li>
    <li role="presentation"><a href="/Team:Lund/Design/Circuit">Circuit</a></li>
    <li role="presentation"><a href="/Team:Lund/Design/Experiments">Experiments</a></li>
    <li role="presentation"><a href="/Team:Lund/Design/References">References</a></li>
  </ul>
</div>

<div class="content-wrapper under-nav text-layout">
  <div class="container">

    <div class="row">
      <div class="col-md-8">
        <section>
          <h3 class="section-heading">Designing our system</h3>
          <p>A novel biosensor for the detection of microplastics was realized using a top-down black-box approach. Microplastics were identified as the target input and a reporter protein was designated as the output, as depicted in <i>fig. 1</i>. Due to their unreactive nature and varied composition, microplastics were quickly deemed a difficult set of molecules to identify using biological sensing elements, calling for an indirect method of identification if tools of synthetic biology were to be utilized. As discussed in the <a href="/Team:Lund/Description">project description</a>, successfully developing such a device would offer a rapid and cost-effective solution in contrast to the arduous filtration processes that are used today.</p>

          <p>Expanding on the groundwork put forward by the UCL’12 iGEM team, establishing the presence of molecules associated with water-bound microplastics was determined to be a viable approach [9]. Two classes of molecules were identified as being commonly associated with microplastics – plasticizers and organic pollutants. They are both lipophilic compounds, allowing trans-membrane diffusion and subsequent intracellular recognition. <em>Escherichia coli </em>was chosen as a functional chassis for the biosensor; not only due to its compatibility with previously developed parts and devices but also as facilitated transmembrane transport of phthalates and organic pollutants has been reported through the FadL and subsequent FadD channels [33] [34]. The system updates are reflected in <i>fig. 2</i>.</p>

          <p>Homology analysis of the two nominated compounds and natural ligands to existing sensory elements generated two transcriptional regulators – NahR with affinity for the most common moieties found in organic pollutants and the human estrogen receptor alpha with noted antagonistic interaction with plasticizers [9] [24]. Both <em>NahR</em> and <em>hER-α</em> serve as transactivators in presence of respective cofactors [15] [27]. Both the full-length hER-α and truncated versions have been successfully transformed into <em>E. coli</em> with adequate expression rates, even if the former has shown toxicity at higher expression rates, and NahR has been utilized as an inducible transcriptional factor in multiple iGEM projects [9] [35] [36] [37] [38] [39]. Thus, they were chosen as the sensory elements of our circuit, as shown in <i>fig. 3</i>.</p>

          <p>To increase the selectivity of the sensor and thus the accuracy of the output readings, the input signals corresponding to the sensing of phthalates and organic pollutants must transform according to that of a logic AND-gate, as illustrated in <i>fig. 4</i>. That is to say, the reporter will only be measureable in the instance that phthalates and organic pollutants are both present in the sample. In an attempt to counteract the potential toxicity of introducing and expressing the full-length <em>hER-α</em> in <em>E. coli</em>, the truncated version hER-α LBD was finalized as the phthalate-sensing element to be applied in a complementation assay through domain insertion, in accordance with previous prosperous <em>hER-α </em>biosensor studies as noted in the <em>theory </em>section [29] [30] [31].</p>

          <p>Employing the conformational change of the hER-α LBD in a <em>genetic</em> AND-gate can be achieved through a plethora of different approaches. However, in each instance a fusion protein needs to be constructed that capitalizes on the ligand-induced convergence of the termini. Furthermore, rather than connecting the <em>NahR</em> and the <em>hER-α LBD</em> in series in the circuit architecture, implementing their expression concurrently would decrease the reporter synthesis response time. These conditions were met using a tripartite split GFP-protein, as schematically shown in <i>fig. 5</i>. Two GFP gene residues, corresponding to beta-sheet 10 and 11 of the GFP beta barrel motif, were attached to each respective side of the <em>hER-α LBD</em> with appropriate linkers. The remaining GFP, <em>GFP1-9</em>, was put downstream <em>Psal</em> – under regulation of NahR. <em>Cabantous</em> <em>et al</em>. iteratively mined the optimal split sites of the GFP for spontaneous self-assembly upon association [40]. Thus  , upon expression of the GFP10- hER-α LBD-GFP11 and GFP1-9, and subsequent antagonistic ligand interaction with the hER-α LBD, self-assembly of the fluorophore ought to occur. Similar results have been reported with a two-split GFP hER-α LBD fusion protein [30] [41]. The added benefit of utilizing a tripartite split GFP with smaller GFP-tags affixed to the hER-α LBD is that somewhat equidistantly split proteins have generally shown either high levels of aggregation or high background noise [30].</p>

          <p>To alleviate the host of constant expression of the heterologous proteins, both <em>nahR</em> and the <em>GFP10- hER-α LBD-GFP11</em> were put downstream <em>lac o</em> with the <em>lac</em> repressor <em>lacI</em> constitutively expressed. Thus, the system remains repressed until addition of a ligand inducing an allosteric shift and subsequent disassociation of the repressor [42]. The final system model can be found in <i>fig. 6</i>.</p>

        </section>
        <section>
          <h3 class="section-heading">Design of fusion protein linkers</h3>
          <p>The choice of linkers was evaluated using the open-source software PyMOL. Crystal structures of the human estrogen receptor alpha were acquired at the NCBI gene bank [43] [44]. The sequence of ligand-free hER-α LBD and antagonist-hER-α LBD complex were analyzed and the distance between the two termini was measured using the PyMOL <em>wizard</em> tool. The former measured a total distance of approximately 38 Å with clear steric hindrance while the latter approximately 21 Å with limited to no steric hindrance. Thus, only linkers longer than 11 Å were considered adequate candidates. Ultimately, glycine-serine linkers were chosen due to their flexibility, which was needed for mobility of the two GFP fragments [45] [46]. Two different linker sizes were picked, corresponding to 6 a.a and 12 a.a respectively. </p>

          <figure>
            <a href="#" data-featherlight="http://2017.igem.org/wiki/images/6/63/T--lund--freereceptorpymol.png">
              <img class="img-responsive center-block" src="http://2017.igem.org/wiki/images/6/63/T--lund--freereceptorpymol.png">
            </a>
            <figcaption>Figure 7: A model of the hER-α LBD without an associated ligand. The 3D protein structure was drawn using the PyMOL software. As depicted, there is a clear steric hindrance between the N- and the C-terminal with a total distance of approximately 38 Å.</figcaption>
          </figure>
          <figure>
            <a href="#" data-featherlight="http://2017.igem.org/wiki/images/4/42/T--lund--antagonstreceptorpymol.png">
              <img class="img-responsive center-block" src="http://2017.igem.org/wiki/images/4/42/T--lund--antagonstreceptorpymol.png">
            </a>
            <figcaption>Figure 8: A model of the hER-α LBD without an associated ligand. The 3D protein structure was drawn using the PyMOL software. There is no obvious steric hindrance between the N- and the C-terminal with a total distance of approximately 21 Å.</figcaption>
          </figure>
        </section>


        </div>
        <div class="col-md-4">
          <figure>
            <a href="#" data-featherlight="http://2017.igem.org/wiki/images/0/0c/T--Lund--bb1.svg">
              <img class="img-responsive center-block img-thumbnail" src="http://2017.igem.org/wiki/images/0/0c/T--Lund--bb1.svg">
            </a>
            <figcaption class="text-center">Fig. 1: System model 1, input signal defined.</figcaption>
          </figure>

          <figure>
            <a href="#" data-featherlight="http://2017.igem.org/wiki/images/7/7b/T--Lund--bb2.svg">
              <img class="img-responsive center-block img-thumbnail" src="http://2017.igem.org/wiki/images/7/7b/T--Lund--bb2.svg">
            </a>
            <figcaption class="text-center">Fig. 2: System model 2, input characteristics specified, chassi chosen.</figcaption>
          </figure>

          <figure>
            <a href="#" data-featherlight="http://2017.igem.org/wiki/images/a/a6/T--Lund--bb13.svg">
              <img class="img-responsive center-block img-thumbnail" src="http://2017.igem.org/wiki/images/a/a6/T--Lund--bb13.svg">
            </a>
            <figcaption class="text-center">Fig. 3: System model 3, sensory components specified.</figcaption>
          </figure>

          <figure>
            <a href="#" data-featherlight="http://2017.igem.org/wiki/images/a/ad/T--Lund--bb4.svg">
              <img class="img-responsive center-block img-thumbnail" src="http://2017.igem.org/wiki/images/a/ad/T--Lund--bb4.svg">
            </a>
            <figcaption class="text-center">Fig. 4: System model 4, output characteristics specified. For the sake of simplicity, ER-&alpha; will be used synonymously with ER-&alpha;-LBD.</figcaption>
          </figure>

          <figure>
            <a href="#" data-featherlight="http://2017.igem.org/wiki/images/8/85/T--Lund--bb5.svg">
              <img class="img-responsive center-block img-thumbnail" src="http://2017.igem.org/wiki/images/8/85/T--Lund--bb5.svg">
            </a>
            <figcaption class="text-center">Fig. 5: System model 5, sensory component relation specified.</figcaption>
          </figure>

          <figure>
            <a href="#" data-featherlight="http://2017.igem.org/wiki/images/d/d6/T--Lund--bb6.svg">
              <img class="img-responsive center-block img-thumbnail" src="http://2017.igem.org/wiki/images/d/d6/T--Lund--bb6.svg">
            </a>
            <figcaption class="text-center">Fig. 6: System model 6, complete genetic circuit.</figcaption>
          </figure>
      </div>
    </div>

  </div>
</div>

</html>
{{Lund/footer}}
