{{Lund}}
<html>

<script type="text/javascript" src="/Team:Lund/scripts/pv?action=raw&ctype=text/javascript"></script>
<script type="text/javascript">
$(document).ready(function() {
  var options = {
    width: 360,
    height: 240,
    antialias: true,
    quality : 'medium'
  };

  var viewer1 = pv.Viewer(document.getElementById('viewer-NahR'), options);
  // Public url: http://2017.igem.org/wiki/images/2/27/T--Lund--NahR.txt
  pv.io.fetchPdb('http://2017.igem.org/wiki/images/2/27/T--Lund--NahR.txt', function(structure) {
    if (structure) {
      mol.assignHelixSheet(structure); // See https://github.com/biasmv/pv/issues/169
      viewer1.cartoon('NahR', structure, { color: pv.color.ssSuccession() });
      viewer1.fitTo(structure);
      viewer1.spin(true);
    }
    // viewer1.fitParent()
  });

  var viewer2 = pv.Viewer(document.getElementById('viewer-hER'), options);
  // Public url: http://2017.igem.org/wiki/images/8/84/T--Lund--hER.txt
  pv.io.fetchPdb('http://2017.igem.org/wiki/images/8/84/T--Lund--hER.txt', function(structure) {
    if (structure) {
      viewer2.cartoon('hER', structure, { color: pv.color.ssSuccession() });
      viewer2.centerOn(structure);
      viewer2.fitTo(structure);
      viewer2.spin(true);
    }
  });
});
</script>

<div id="design-page">

<header class="design-landing">
  <div class="container">
    <h1 class="design-landing-texts ">Design</h1>
  </div>
</header>

<div class="container">
  <ul class="nav nav-pills nav-auto-stacked">
    <li role="presentation"><a href="/Team:Lund/Design">Overview</a></li>
    <li role="presentation"><a href="/Team:Lund/Design/Applications">Applications</a></li>
    <li role="presentation" class="active"><a href="/Team:Lund/Design/Theory">Theory</a></li>
    <li role="presentation"><a href="/Team:Lund/Design/Design">Design</a></li>
    <li role="presentation"><a href="/Team:Lund/Design/Experiments">Experiments</a></li>
    <!-- <li role="presentation"><a href="/Team:Lund/Design/References">References</a></li> -->
  </ul>
</div>

<div class="content-wrapper under-nav text-layout">
  <div class="container">
    <div class="row">
      <div class="col-md-8">
        <h3 class="section-heading">Introduction</h3>

        <p>
          <i><i>Vitreoscilla</i></i> hemoglobin (VHb) is a protein found in the aerobic gram negative bacteria <i><i>Vitreoscilla</i> spp.</i> It is regulated by the <i><i>Vitreoscilla</i></i> hemoglobin promoter, which is oxygen dependent and activates the transcription of the hemoglobin under microaerobic conditions to increase oxygen uptake [1]. It was first discovered in 1966 and was initially believed to be a  cytochrome, until its primary structure was disclosed in 1986 and it was recognized as a hemoglobin [3]. The gene coding for VHb (<i>vgb</i>) was cloned for the first time and expressed in <i>Escherichia coli</i> in 1988, and its capability to increase oxygen uptake was quickly noted [4][5].
        </p>
        <p>
          The discovery led to a new approach in combating the reduced yields in fermentors caused by low oxygen conditions arising from high density cell culture: instead of modifying external parameters in order to increase oxygen availability, hemoglobin is co-expressed in order to enhance the cellular oxygen utilization. In this way, the cells are able to aerobically catabolize the available carbon sources under microaerobic conditions, without generating growth-inhibitory metabolites such as lactate and acetate [5]. Furthermore, it has been found that in addition of being able to increase protein yields under microaerobic conditions, the presence is also correlated with improved cell growth and survivability [5]. This seems to be the case not only for <i><i>Vitreoscilla</i></i> and <i>E. coli</i>, but also for other bacterial species [6], yeast [8], plants [9] and even some vertebrates [10].
        </p>
      </div>
    </div>

    <div class="row">
      <div class="col-md-8">
        <h3 class="section-heading">Role in oxygen delivery</h3>
        <p>It seems that the way in which VHb enhances oxygen uptake is by acting as an oxygen transporter, directly binding oxygen and delivering it to the respiratory chain. This claim is supported by several studies covering different mechanisms of VHb. More specifically, there are two key observations that motivate this. </p>

        <p>First, it has been shown <i>in vitro</i> that VHb has a high binding affinity to lipid monolayers extracted from <i>E. coli</i> and that once bound, the oxygen affinity is reduced by more than 20-fold [11]. This is believed to be caused by the hydrophobic region between the CE-corner and the E-helix at the distal pocket serving as an anchor to the acyl chains of the phospholipid bilayer of the cell membrane. Due to the interaction between VHb and the membrane, a conformational change would occur around the heme pocket and the oxygen binding affinity would immediately be reduced. </p>

        <p>Secondly, it has been shown by immunogold labeling in electron microscopy that VHb is localized in the cytoplasm and to a large extent near the cell membrane in both </i><i>Vitreoscilla</i></i> and <i>E. coli</i> [12]. These two observations indicate that VHb indeed transports oxygen to the respiratory chain, since if VHb is not in proximity to the cell membrane, it has high affinity for oxygen and <i>vice versa</i>, if bound the affinity is greatly reduced. Since respiration occurs around cell membranes, it seems highly plausible that VHb transports oxygen to the respiratory chain via direct interaction.</p>

        <p>Other supporting experiments include showing that VHb can stimulate oxygenase activity [13][14] and even itself serve as a terminal oxidase [15]. The latter was shown by cultivating cells with no terminal oxidases under aerobic conditions, in which only cells containing VHb could maintain an aerobic respiration. Moreover, it has been shown that VHb protects from oxidative stress [16][17] and that increased expression levels are correlated with increased cell growth [18]. </p>


        <p>
          The putative mechanism is illustrated in <i>fig. 1</i>. When VHb is free in the cytoplasm, it has a high affinity to oxygen and binds accordingly. Then, anchoring to the cell membrane causes a decrease in affinity and a subsequent oxygen release. This oxygen may then be consumed within the respiratory chain.
        </p>

        <figure>
          <a href="#" data-featherlight="http://2018.igem.org/wiki/images/1/1f/T--Lund--vhb_lipid.png">
            <img class="img-responsive center-block img-thumbnail" src="http://2018.igem.org/wiki/images/1/1f/T--Lund--vhb_lipid.png">
          </a>
          <figcaption>Figure 1: VHb binds to membrane and releases oxygen</figcaption>
        </figure>
      </div>
    </div>

    <div class="row">
      <div class="col-md-8">
        <h3 class="section-heading">Structure</h3>

        <p>
          VHb is a homodimeric protein, consisting of two subunits of 146 amino acid residues each and with a close resemblance to the globin fold [2]. The globin fold is a three-dimensional motif shared among the members of the globin superfamily, consisting of six to eight alpha-helices packed at an approximate 50 degree angle from each other. Starting from the N-terminus, the helices are denoted as helix A-H. Between each helix and the following one is a loop connecting the segments, denoted by AB, BC, CD etc., corresponding to each pair of helices. The pocket in which heme is located is formed by a hydrophobic cavity, created by the distal E-helix and proximal F-helix, in  which the conserved proximal histidine F8 coordinates with the upper portion of the heme group.           </p>

        <p>
          The VHb monomer is illustrated in <i>fig. 2</i> along with the sperm whale myoglobin (Mb) which is commonly considered as the reference monomeric hemoglobin. The globin fold is clearly visible and the similarities are apparent, with the heme residing in the pocket enclosed by the E and F alpha helices. However, while the globin fold of Mb consists of eight alpha helices, the VHb has only seven as there is no apparent structure in the expected D-helix region [19]. This gives rise to a more hydrophobic CE transition in contrast to the usual CD, which along with the initial hydrophobic and disordered E7-E10 segment of the E helix is believed to be one of the key reasons for the high lipid binding propensity of VHb [11][19].
      </div>

      <div class="col-md-4">
        <figure>
          <a href="#" data-featherlight="http://2018.igem.org/wiki/images/1/1f/T--Lund--vhb_lipid.png">
            <img class="img-responsive center-block img-thumbnail" src="http://2018.igem.org/wiki/images/1/1f/T--Lund--vhb_lipid.png">
          </a>
          <a href="#" data-featherlight="http://2018.igem.org/wiki/images/1/1f/T--Lund--vhb_lipid.png">
            <img class="img-responsive center-block img-thumbnail" src="http://2018.igem.org/wiki/images/1/1f/T--Lund--vhb_lipid.png">
          </a>
          <figcaption>Figure 2: VHb binds to membrane and releases oxygen</figcaption>
        </figure>
      </div>
    </div>

    <div class="row">
      <div class="col-md-8">
        <h3 class="section-heading">Previous research</h3>
        <p>
          Several studies have investigated the influence of the VHb expression level on the cell growth and the production of various desired products. In 2010, Sanny <i>et al.</i> managed to increase production of ethanol in <i>E. coli</i> by as much as 119% when expressing VHb in their ethanol-producing strain. However, they found that when they increased the VHb expression level by inserting the <i>vgb</i> gene into other plasmids, the ethanol production decreased to levels below that of the strain with no VHb expression at all, indicating that only low levels of VHb expression were beneficial [20].
        </p>

        <p>
          Yu <i>et al.</i> studied the effect of VHb on the production of the biopolymer poly(β-hydroxybutyrate) (PHB) in <i>E. coli</i> both by integrating the <i>vgb</i> gene into the host chromosome and by keeping it on a plasmid. What they found was that chromosomal integration if <i>vgb</i> increased PHB content in cells grown in flasks to 83.8%, as compared to the non-vgb carrying strain where the content was less than 70% under the same conditions. However, when the <i>vgb</i> gene was instead carried on a plasmid, both PHB production and cell growth were far lower than with no VHb at all [21]. They theorized that the negative effect might be due to the burden of carrying an additional or a larger plasmid to accommodate the <i>vgb</i> gene. However, it has been shown that integrating <i>vgb</i> on the chromosome of the <i>E. coli</i> strain in question lowers the expression level compared to when a multicopy plasmid is used [22], again suggesting that a lower expression level may be more beneficial than a high one.
        </p>

        <p>
          In both the studies described above, VHb was shown to only be beneficial when expressed at a low level, with higher VHb expression leading to a poorer outcome than with no VHb at all. This is in contrast to a study done in 1996 by Tsai <i>et al.</i>, who by modulating the concentration of the inducer Isopropyl β-D-1-thiogalactopyranoside (IPTG) showed that increasing levels of VHb expression was associated with increasing final cell density (up to 2.7-fold compared to the VHb- strain) and a reduction in fermentation byproducts such as acetate and lactate [18]. However, as opposed to the other two groups above, they did not study the formation of any specific industrially relevant product.
        </p>

        <p>The three studies described above all employed different methods to obtain variation in the VHb expression level - changing the IPTG concentration [18], inserting the <i>vgb</i> gene into different plasmids [20] and chromosomal integration [21]. However, we have not found any articles studying the effect on yield or productivity where the only variable in the expression system is the strength of the promoter under which VHb is expressed. In addition, VHb seems underutilized, especially within iGEM. We wanted to investigate its relevance in the synthetic biology community. </p>
      </div>
    </div>

    <div class="row">
      <div class="col-md-8">
        <h3 class="section-heading">Previous iGEM teams </h3>
        <p>
          Until now, only two previous iGEM teams have to our knowledge adopted VHb technology in their projects. Imperial college 2014 co-expressed <i>vgb</i> in <i>Gluconacetobacter xylinus</i> to promote the production of cellulose. By doing so, they achieved an almost twofold increase in cell density using the <i>G. xylinus</i> iGEM strain. SCU-China 2016 used the same gene in <i>E. coli</i> to enhance growth in oxygen limited environments. It was shown that by expressing <i>vgb</i>, an improvement of both growth and protein production was obtained in the early stage of cell cultivation. However, no consistent improvements were observed in the stationary cultivation phase.
        </p>
      </div>
    </div>
    <div class="row">
      <div class="col-md-8">
        <h3 class="section-heading">References</h3>
        <button id="refBtn" class="btn btn-default" type="button" data-toggle="collapse" data-target="#ref" aria-expanded="false" aria-controls="ref">
          Show
        </button>
        <div class="collapse" id="ref">
          <ol class="references">
            <li>[1] Dikshit, K., Spaulding, D., Braun, A., and Webster, D. (1989) Oxygen Inhibition of Globin Gene Transcription and Bacterial Haemoglobin Synthesis in <i><i>Vitreoscilla</i></i>. <i>Microbiology 135</i>, 2601-2609.</li>
            <li>[2] Stark, B., Dikshit, K., and Pagilla, K. (2012) The Biochemistry of <i><i>Vitreoscilla</i></i> hemoglobin. <i>Computational and Structural Biotechnology Journal 3</i>, e201210002.</li>
            <li>[3] Wakabayashi, S., Matsubara, H., and Webster, D. (1986) Primary sequence of a dimeric bacterial haemoglobin from <i><i>Vitreoscilla</i></i>. <i>Nature 322</i>, 481-483.</li>
            <li>[4] Dikshit, K., and Webster, D. (1988) Cloning, characterization and expression of the bacterial globin gene from <i><i>Vitreoscilla</i></i> in <i>Escherichia coli</i>. <i>Gene 70</i>, 377-386.</li>
            <li>[5] Khosla, C., and Bailey, J. (1988) Heterologous expression of a bacterial haemoglobin improves the growth properties of recombinant <i>Escherichia coli</i>. <i>Nature 331</i>, 633-635.</li>
            <li>[6] Wang S, Liu F, Hou Z, Zong G, Zhu X, Ling P. Enhancement of natamycin production on <i>Streptomyces gilvosporeus</i> by chromosomal integration of the <i><i>Vitreoscilla</i></i> hemoglobin gene (<i>vgb</i>) <i>World J Microb Biot. 2014;30</i>:1369–1376. doi: 10.1007/s11274-013-1561-4</li>
            <li>[8] Wu, JM. and Fu, WC. (2012) Intracellular co-expression of <i><i>Vitreoscilla</i></i> hemoglobin enhances cell performance and β-galactosidase production in <i>Pichia pastoris</i>. <i>J Biosci Bioeng 113(3)</i>, 332–337.</li>
            <li>[9] Holmberg, N., Lilius, G., Bailey, J., and Bülow, L. (1997) Transgenic tobacco expressing <i><i>Vitreoscilla</i></i> hemoglobin exhibits enhanced growth and altered metabolite production. <i>Nature Biotechnology 15</i>, 244-247.</li>
            <li>[10] Guan, B., Ma, H., Wang, Y., Hu, Y., Lin, Z., Zhu, Z., and Hu, W. (2010) <i><i>Vitreoscilla</i></i> Hemoglobin (VHb) Overexpression Increases Hypoxia Tolerance in Zebrafish (<i>Danio rerio</i>). <i>Marine Biotechnology 13</i>, 336-344.</li>
            <li>[11] Rinaldi A. C., Bonamore A., Macone A., Boffi A., Bozzi A., Di Giulio A. (2006) Interaction of <i><i>Vitreoscilla</i></i> hemoglobin with membrane lipids. <i>Biochemistry 45</i>:4069–4076</li>
            <li>[12] Ramandeep D., Hwang K. W., Raje M., Kim K.J., Stark B.C., Dikshit K.L., Webster D. A. (2001) <i><i>Vitreoscilla</i></i> hemoglobin: intracellular localization and binding to membranes. <i>J. Biol. Chem. 277</i>:24781–24789.</li>
            <li>[13] Fish P. A., Webster D. A., Stark B. C. (2001) <i><i>Vitreoscilla</i><i> hemoglobin enhances the first step in 2,4-dinitrotoluene degradation <i>in vitro</i> and at low aeration in vivo. <i>J. Mol. Catal. B. Enzym. 9</i>:75–82.</li>
            <li>[14] Lin J. M., Stark B. C., Webster D. A. (2003) Effects of <i>Vitreoscilla</i> hemoglobin on the 2,4-dinitrotoluene (DNT) dioxygenase activity of <i>Burkholderia</i> and on DNT degradation in two-phase bioreactors. <i>J. Ind. Microbiol. Biotechnol. 30</i>:362–368</li>
            <li>[15] Dikshit R. P., Dikshit K. L., Liu Y., Webster D. A. (1992) The bacterial hemoglobin from <i>Vitreoscilla</i> can support the aerobic growth of E. coli lacking terminal oxidases. <i>Arch. Biochem. Biophys.</i> 293:241–245.</li>
            <li>[16] Geckil H., Gencer S., Kahraman H., Erenler S. O.(2003) Genetic engineering of <i>Enterobacter aerogenes</i> with the <i>Vitreoscilla</i> hemoglobin gene: cell growth, survival, and antioxidant enzyme status under oxidative stress. <i>Res. Microbiol.</i> 154:425–431.</li>
            <li>[17] Kvist M., Ryabova E. S., Nordlander E., Bulow L. (2007) An investigation of the peroxidase activity of <i>Vitreoscilla</i> hemoglobin. <i>J. Biol. Inorg. Chem.</i> 12:324–334.</li>
            <li>[18] Tsai PS, Hatzimanikatis V, Bailey JE. (1996). Effect of <i>Vitreoscilla</i> Hemoglobin Dosage on Microaerobic <i>Escherichia coli</i> Carbon and Energy Metabolism. <i>Biotechnology and Bioengineering, 49</i>, 139-150.</li>
            <li>[19] Tarricone, C., Galizzi, A., Coda, A., Ascenzi, P., and Bolognesi, M. (1997) Unusual structure of the oxygen-binding site in the dimeric bacterial hemoglobin from <i>Vitreoscilla</i> <i>sp.</i> <i>Structure 5</i>, 497-507.</li>
            <li>[20] Sanny, T., Arnaldos, M., Kunkel, S.A. et al (2010). Engineering of ethanolic <i>E. coli</i> with the <i>Vitreoscilla</i> hemoglobin gene enhances ethanol production from both glucose and xylose. <i>Appl Microbiol Biotechnol 88</i>, 1103-1112. https://doi.org/10.1007/s00253-010-2817-7</li>
            <li>[21] Yu, H., Yin, J., Li, H., Yang, S. and Shen, Z. (2000). Construction and Selection of the Novel Recombinant <i>Escherichia coli</i> Strain for Poly(β-Hydroxybutyrate) Production.  <i>J Biosci Bioeng 89(4)</i>, 307-311. </li>
            <li>[22] Wu, Y., and Yang, S. (1996) Construction and Study of an Integrated Expression Vector of <i>Vitreoscilla</i> Hemoglobin Gene (vgb) by Double Homologous Recombination Between Plasmid and Chromosome. <i>Chinese Journal of Biotechnology 12</i>, 276-283.  </li>
          </ol>
        </div>
      </div>
    </div>
    <!-- <section>
      <div class="row">
        <div class="col-md-8">
          <h3 class="section-heading">Sensing organic pollutants</h3>
        </div>
      </div>
      <div class="row">
        <div class="col-md-8">
          <p>Persistent organic pollutants are harmful organic compounds of anthropogenic origins that have been, and in some cases still are, utilized as industry chemicals and pesticides. Due to their ecotoxicity and their propensity of bioaccumulation, the United Nations Environment Programme (UNEP) adopted and implemented proposed regulation of these chemicals as put forward by the Stockholm Convention on Persistent Organic Pollutants in 2001 [10]. As of 2017, 181 countries have ratified the convention [11]. While there is no explicit organic backbone, most organic pollutants share certain characteristics. They have one or more cyclic ring structures, most often of the aromatic nature, they are highly lipophilic and they are, to a varied extent, halogenated [12]. Consequently, there are different moieties that can be exploited when searching for homologues that act as natural ligands to gene regulatory elements. The 2012 UCL iGEM team successfully determined and utilized the homology between the NahR inducing ligand salicylate and the aromatic backbone found in most organic pollutants [9]. </p>
          <p>The <em>NahR</em> gene encodes a Lys-R type transcription regulator found in the naphthalene degrading operon of the NAH7 plasmid of various <em>Pseudomas</em> [13]. As with all Lys-R type regulators, NahR has a conserved N-terminal DNA-binding helix-turn-helix motif and a C-terminal co-inducer-binding domain [14]. The catabolic genes are organized in two operons, <em>fig. 1</em>, <em>nah</em> (<em>nah A-F) </em>and <em>sal</em> (<em>sal G-M</em>) that encode enzymes for metabolism of naphthalene to salicylate, and salicylate to the TCA-intermediates pyruvate and acetaldehyde respectively [13]. NahR regulates the expression of both <em>sal </em>and <em>nah</em> through binding 60 bp upstream of each respective promoter, <em>Psal </em>and<em> Pnah</em>, and is constitutively expressed upstream <em>nah-G</em> [15]<em>.</em> It binds regardless of the presence of the inducer salicylate. However, the transcriptional induction rate is approximately 20 times higher upon association with salicylate, as this induces a conformational change in the DNA-NahR complex that relieves DNA-bending at the site and subsequently allows better DNA-RNAP interaction [16]. The conformational change springs from an additional interaction between the NahR and the DNA 35 bp upstream of the RNAP binding site [15].</p>
          <p>There lies some uncertainty in the exact configuration of the DNA-NahR association. Some reports suggest that NahR binds as a monomer, while other research indicates multimerization at the binding site. Moreover, it remains unknown whether the binding is cooperative or not [15]. For further elaboration on the issue, see <a href="/Team:Lund/Model">modeling</a>.</p>
        </div>
        <div class="col-md-4">

          <figure>
            <a href="#" data-featherlight="http://2017.igem.org/wiki/images/d/d2/T--Lund--naphoperon.png">
              <img class="img-responsive center-block img-thumbnail" src="http://2017.igem.org/wiki/images/d/d2/T--Lund--naphoperon.png">
            </a>
            <figcaption>Figure 1: Schematic representation of the organization of the upper and lower degradation pathway of naphthalene found in the Nah7 plasmid of <em>Pseudomas</em>. Both pathways are under regulation of the constitutively expressed regulator NahR [17].</figcaption>
          </figure>

          <figure>
            <div id="viewer-NahR"></div>
            <figcaption>Figure 2: 3D structure of the NahR, a transcriptional regulator spanning 306 amino acids. The protein structure is organized as follows; the N-terminal consists of a highly conserved DNA-binding helix-turn-helix motif. The DNA association is furthermore dependent on a multimerization site located along the C-terminal. Through iterative mutagenesis, the ligand-binding pocket has been determined to lie close to equidistantly between the termini [18] [47] [48] .</figcaption>
          </figure>
        </div>
      </div>
    </section>
    <section>
      <div class="row">
        <div class="col-md-8">
          <h3 class="section-heading">Sensing plasticizers</h3>
        </div>
      </div>
      <div class="row">
        <div class="col-md-8">
          <p>Plasticizers are very common additives found in most commercial plastic products. They provide the polymers with durability, elasticity and flexibility, offering increased malleability and strength [19]. The plasticizer action on the polymeric material has generally thought to be lubrication between the polymeric chains, but recent thermodynamic simulations have shown that the situation might be more complex [20]. The most predominant class of plasticizers used today is that of low-weight phthalate esters (PEs), in particular DEHP (bis-2-ethylhexyl phthalate) [19]. While PEs offer a cost-effective solution to maintaining flexibility, there is cause for concern as they are not covalently bonded to the plastic and will readily leach into the environment. Multiple sources have reported severe adverse effects in vertebrates; among other things, toxic action on the human endocrine system has been noted [21] [22] [23] [24] [25]. In particular, low-weight PEs have demonstrated some antagonistic affinity for the ligand-binding domain of the human estrogen-alpha receptor (hER-α) [23] [24], making it a viable candidate for detecting plasticizers.</p>
          <p>The estrogen receptor is a member of the intracellular receptor family and exists in two different forms, hER-α and hER-β, with significant overlapping sequence homology. Each receptor shares a common architecture consisting of five domains; the N-terminal <em>A/B</em> region able to regulate gene transcription in absence of a ligand, the <em>C</em>-domain able to bind to a designated DNA sequence (DBD) and a hinge region <em>D</em> (DBD) that connects the C-domain and the ligand binding domain (LBD) <em>E</em> [26]. The mechanism of gene transactivation by the estrogen receptor has been studied in detail through analysis of the ligand-induced conformational change of the LBD. The LBD has a similar organization to that of the other nuclear receptor LBDs, with a three-layered antiparallel α-helical sandwich motif, for a total of 11 α-helices (H1-11) with one additional anti-parallel β-sheet flanking the arrangement (S1-2). The central core, located between the two outer helical layers, constitutes a wedge-shaped molecular scaffold to bind the ligand [27]. Upon interaction between the binding site and a ligand, a sizeable conformational change occurs at the C-terminal helix H12 to modify the action at the DNA-interface. An agonistic ligand spatially displaces H12 to fit snugly over the cavity whereas an antagonistic ligand prevents such confirmation and instead forces the helix to reposition through a 130° degree rotation toward the N-terminus [27][28]. Such displacement of H12 has been utilized in complementation assays as a <em>molecular switch</em>. Complementation of a split protein fused to each respective side of a truncated hER-α LBD is made possible upon antagonist association with the binding site as the termini are subsequently brought in close proximity to one another [29] [30] [31].</p>

        </div>
        <div class="col-md-8 col-md-4">
          <figure>
            <div id="viewer-hER"></div>
            <figcaption>Figure 3: 3D structure of the human estrogen receptor alpha ligand-binding domain (in complex with estradiol), a domain found in the transcriptional transactivator hER-α spanning 300 amino acids. The ligand-binding domain consists of a deep and promiscuous ligand-binding pocket consisting of a three-layered antiparallel α-helical sandwich motif. Upon recognition of a ligand, helix H12 will change conformation in accordance with the nature of the ligand; antagonists will displace the helix and bring the two termini close to one another whereas agonists will induce helix H12 to transverse the protein and stabilize over the pocket [27] [49] [50] [51] [52] .</figcaption>
          </figure>

          <figure>
            <a href="#" data-featherlight="http://2017.igem.org/wiki/images/1/1a/T--Lund--something.png">
              <img class="img-responsive center-block img-thumbnail" src="http://2017.igem.org/wiki/images/1/1a/T--Lund--something.png">
            </a>
            <figcaption>Figure 4: The hER-a ligand binding has been implemented and utilized successfully in complementation assays through capitalizing on the conformational change upon ligand recognition. [32]</figcaption>
          </figure>

        </div>
      </div>
    </section> -->

  </div>
</div>
</div>
</html>
{{Lund/footer}}
