{{Lund}}
<html>

<script type="text/javascript" src="/Team:Lund/scripts/pv?action=raw&ctype=text/javascript"></script>
<script type="text/javascript">
$(document).ready(function() {
  var options = {
    width: 360,
    height: 240,
    antialias: true,
    quality : 'medium'
  };

  var viewer1 = pv.Viewer(document.getElementById('viewer-NahR'), options);
  // Public url: http://2017.igem.org/wiki/images/2/27/T--Lund--NahR.txt
  pv.io.fetchPdb('http://2017.igem.org/wiki/images/2/27/T--Lund--NahR.txt', function(structure) {
    if (structure) {
      mol.assignHelixSheet(structure); // See https://github.com/biasmv/pv/issues/169
      viewer1.cartoon('NahR', structure, { color: pv.color.ssSuccession() });
      viewer1.fitTo(structure);
      viewer1.spin(true);
    }
    // viewer1.fitParent()
  });

  var viewer2 = pv.Viewer(document.getElementById('viewer-hER'), options);
  // Public url: http://2017.igem.org/wiki/images/8/84/T--Lund--hER.txt
  pv.io.fetchPdb('http://2017.igem.org/wiki/images/8/84/T--Lund--hER.txt', function(structure) {
    if (structure) {
      viewer2.cartoon('hER', structure, { color: pv.color.ssSuccession() });
      viewer2.centerOn(structure);
      viewer2.fitTo(structure);
      viewer2.spin(true);
    }
  });
});
</script>

<header class="design-header">
  <div class="container">
    <h1 class="display">Design</h1>
  </div>
</header>

<div class="container">
  <ul class="nav nav-pills nav-auto-stacked">
    <li role="presentation"><a href="/Team:Lund/Design">Overview</a></li>
    <!-- <li role="presentation"><a href="/Team:Lund/Design/Approach">Approach</a></li> -->
    <li role="presentation" class="active"><a href="/Team:Lund/Design/Theory">Theory</a></li>
    <li role="presentation"><a href="/Team:Lund/Design/Design">Design</a></li>
    <li role="presentation"><a href="/Team:Lund/Design/Experiments">Experiments</a></li>
    <li role="presentation"><a href="/Team:Lund/Design/References">References</a></li>
  </ul>
</div>

<div class="content-wrapper under-nav text-layout">
  <div class="container">
    <div class="row">
      <div class="col-md-8">
        <h3 class="section-heading">Introduction</h3>

        <p>
          Vitreoscilla hemoglobin (VHb) is a protein found in the aerobic gram negative bacteria Vitreoscilla spp. It is regulated by the vitreoscilla hemoglobin promoter which is oxygen dependent and activates the transcription of the hemoglobin under microaerobic conditions to increase oxygen uptake [1]. It was first discovered in 1966 and was initially regarded a cytochrome until the disclosure of its primary structure in 1986 of which it was immediately recognized as a hemoglobin [3]. Shortly after the gene (vgb) was for the first time cloned and expressed recombinantly in Escherichia coli  (E. coli) in 1988, its capability to increase oxygen uptake was immediately noted [4][5].
        </p>
        <p>
          The discovery started a new approach in combating the reduced yields in fermentors caused by low oxygen conditions arising from high density cell cultures. Instead of modifying environmental parameters in order to increase oxygen availability, hemoglobin is instead co-expressed in order to enhance the cellular oxygen utilization. In this way, the cells are able to aerobically catabolize the available carbon sources under microaerobic conditions without generating growth-inhibitory metabolites such as lactate and acetate which is the major bottleneck under these sparse conditions [5]. With this, it has been found that in addition of being able to increase protein yields under microaerobic conditions, the presence is also correlated with improved cell growth and survivability [5]. This seems to be the case for not only Vitreoscilla and E. coli, but also for other bacterial species [6], yeast [8], plants[9] and even some vertebrates [10].
        </p>
      </div>
    </div>

    <div class="row">
      <div class="col-md-8">
        <h3 class="section-heading">Structure</h3>

        <p>
          The protein is a homodimeric hemoglobin, consisting of two 146 amino acid residue subunits with a close resemblance to the globin fold [2]. The globin fold is a three dimensional motif shared among the globin superfamily and consists of six to eight alpha-helices packed in an approximate 50 degree angle. Starting from the N-terminus, each helix is denoted as helix A-H. Between each helix there is a loop connecting the segments, denoted by AB, BC, CD etc., corresponding to each pair of helices. The pocket at which heme is located is formed by a hydrophobic cavity created by the distal E-helix and proximal F-helix by which the conserved proximal histidine F8 coordinates with the upper portion of the heme-group.
        </p>

        <p>
          The VHb monomer is illustrated fig. 2 along with the sperm whale myoglobin (Mb) which is commonly considered as the reference monomeric hemoglobin. The globin fold is clearly visible and the similarities are apparent with the heme residing in the pocket enclosed by the E- and F alpha helices. However, while the globin fold of Mb consists of eight alpha helices, the VHb has only seven as there is no apparent structure in the expected D-helix region [19]. This gives rise to a hydrophobic CE transition in contrast to the usual CD, which along with the initial hydrophobic and disordered E7-E10 segment of the E-helix is believed to one of the key reasons in the high lipid binding propensity of VHb [11][19].
        </p>
      </div>

      <div class="col-md-4">
        <figure>
          <a href="#" data-featherlight="http://2018.igem.org/wiki/images/1/1f/T--Lund--vhb_lipid.png">
            <img class="img-responsive center-block img-thumbnail" src="http://2018.igem.org/wiki/images/1/1f/T--Lund--vhb_lipid.png">
          </a>
          <a href="#" data-featherlight="http://2018.igem.org/wiki/images/1/1f/T--Lund--vhb_lipid.png">
            <img class="img-responsive center-block img-thumbnail" src="http://2018.igem.org/wiki/images/1/1f/T--Lund--vhb_lipid.png">
          </a>
          <figcaption>Figure 2: VHb binds to membrane and releases oxygen</figcaption>
        </figure>
      </div>
    </div>

    <div class="row">
      <div class="col-md-8">
        <h3 class="section-heading">Role in oxygen delivery</h3>

        <p>
          It seems that the way in which VHb enhances oxygen uptake is by acting as an oxygen transporter by directly binding to oxygen and delivering it to the respiratory chain. This claim is supported by several studies covering different mechanisms of VHb. More specifically, there are two key observations that motivate this.
        </p>

        <p>
          First, it has been shown that VHb has a high in vitro binding affinity to lipid monolayers extracted from E. coli and that once bound, the oxygen affinity is reduced by more than 20-fold [11]. This is believed to be caused by the hydrophobic region between the CE-corner and the E-helix at the distal pocket serving as an anchor to the acyl chains of the phospholipid bilayer. By the consequent interaction, a conformational change would occur around the heme pocket and thereby immediately reduce the oxygen binding affinity.
        </p>

        <p>
          Secondly, it has been shown by immunogold labeling in electron microscopy that VHb is localized to the cytoplasm and to a large extent by the cell membrane in both Vitreoscilla and E. coli [12]. These two observations indicate that VHb indeed transports oxygen to the respiratory chain, since if VHb is not in proximity to the cell membrane, it has high affinity for oxygen and vice versa, if bound the affinity is greatly reduced. Now, since respiration occurs around cell membranes, it seems highly plausible that VHb transports oxygen to the respiratory chain via direct interaction.
        </p>

        <p>
          Other supporting experiments includes showing that VHb can stimulate oxygenase activity [13][14] and even itself serve as a terminal oxidase [15]. The latter was shown by letting cells without any terminal oxidases cultivate under aerobic conditions, of which only cells containing VHb could maintain an aerobic respiration. Moreover, it has been shown that VHb protects from oxidative stress [16] [17] and that increased expression levels are correlated with improved cell growth [18].
        </p>

        <figure>
          <a href="#" data-featherlight="http://2018.igem.org/wiki/images/1/1f/T--Lund--vhb_lipid.png">
            <img class="img-responsive center-block img-thumbnail" src="http://2018.igem.org/wiki/images/1/1f/T--Lund--vhb_lipid.png">
          </a>
          <figcaption>Figure 1: VHb binds to membrane and releases oxygen</figcaption>
        </figure>
      </div>
    </div>

    <div class="row">
      <div class="col-md-8">
        <h3 class="section-heading">References</h3>

        <ol class="references">
          <li>[1] Dikshit, K., Spaulding, D., Braun, A., and Webster, D. (1989) Oxygen Inhibition of Globin Gene Transcription and Bacterial Haemoglobin Synthesis in Vitreoscilla. Microbiology 135, 2601-2609.</li>
          <li>[2] Stark, B., Dikshit, K., and Pagilla, K. (2012) The Biochemistry of Vitreoscilla hemoglobin. Computational and Structural Biotechnology Journal 3, e201210002.</li>
          <li>[3] Wakabayashi, S., Matsubara, H., and Webster, D. (1986) Primary sequence of a dimeric bacterial haemoglobin from Vitreoscilla. Nature 322, 481-483.</li>
          <li>[4] Dikshit, K., and Webster, D. (1988) Cloning, characterization and expression of the bacterial globin gene from Vitreoscilla in Escherichia coli. Gene 70, 377-386.</li>
          <li>[5] Khosla, C., and Bailey, J. (1988) Heterologous expression of a bacterial haemoglobin improves the growth properties of recombinant Escherichia coli. Nature 331, 633-635.</li>
          <li>[6] Wang S, Liu F, Hou Z, Zong G, Zhu X, Ling P. Enhancement of natamycin production on Streptomyces gilvosporeus by chromosomal integration of the Vitreoscilla hemoglobin gene (vgb) World J Microb Biot. 2014;30:1369–1376. doi: 10.1007/s11274-013-1561-4</li>
          <li>[8] Wu, JM. and Fu, WC. (2012) Intracellular co-expression of Vitreoscilla hemoglobin enhances cell performance and β-galactosidase production in Pichia pastoris. J Biosci Bioeng 113(3), 332–337.</li>
          <li>[9] Holmberg, N., Lilius, G., Bailey, J., and Bülow, L. (1997) Transgenic tobacco expressing Vitreoscilla hemoglobin exhibits enhanced growth and altered metabolite production. Nature Biotechnology 15, 244-247.</li>
          <li>[10] Guan, B., Ma, H., Wang, Y., Hu, Y., Lin, Z., Zhu, Z., and Hu, W. (2010) Vitreoscilla Hemoglobin (VHb) Overexpression Increases Hypoxia Tolerance in Zebrafish (Danio rerio). Marine Biotechnology 13, 336-344.</li>
          <li>[11] Rinaldi A. C., Bonamore A., Macone A., Boffi A., Bozzi A., Di Giulio A. (2006) Interaction of Vitreoscilla hemoglobin with membrane lipids. Biochemistry 45:4069–4076</li>
          <li>[12] Ramandeep D., Hwang K. W., Raje M., Kim K.J., Stark B.C., Dikshit K.L., Webster D. A. (2001) Vitreoscilla hemoglobin: intracellular localization and binding to membranes. J. Biol. Chem. 277:24781–24789.</li>
          <li>[13] Fish P. A., Webster D. A., Stark B. C. (2001) Vitreoscilla hemoglobin enhances the first step in 2,4-dinitrotoluene degradation in vitro and at low aeration in vivo. J. Mol. Catal. B. Enzym. 9:75–82.</li>
          <li>[14] Lin J. M., Stark B. C., Webster D. A. (2003) Effects of Vitreoscilla hemoglobin on the 2,4-dinitrotoluene (DNT) dioxygenase activity of Burkholderia and on DNT degradation in two-phase bioreactors. J. Ind. Microbiol. Biotechnol. 30:362–368</li>
          <li>[15] Dikshit R. P., Dikshit K. L., Liu Y., Webster D. A. (1992) The bacterial hemoglobin from Vitreoscilla can support the aerobic growth of E. coli lacking terminal oxidases. Arch. Biochem. Biophys. 293:241–245.</li>
          <li>[16] Geckil H., Gencer S., Kahraman H., Erenler S. O.(2003) Genetic engineering of Enterobacter aerogenes with the Vitreoscilla hemoglobin gene: cell growth, survival, and antioxidant enzyme status under oxidative stress. Res. Microbiol. 154:425–431.</li>
          <li>[17] Kvist M., Ryabova E. S., Nordlander E., Bulow L. (2007) An investigation of the peroxidase activity of Vitreoscilla hemoglobin. J. Biol. Inorg. Chem. 12:324–334.</li>
          <li>[18] Tsai PS, Hatzimanikatis V, Bailey JE. (1996). Effect of Vitreoscilla Hemoglobin Dosage on Microaerobic Escherichia coli Carbon and Energy Metabolism. Biotechnology and Bioengineering, 49, 139-150.</li>
          <li>[19] Tarricone, C., Galizzi, A., Coda, A., Ascenzi, P., and Bolognesi, M. (1997) Unusual structure of the oxygen-binding site in the dimeric bacterial hemoglobin from Vitreoscilla sp. Structure 5, 497-507.</li>
      </ol>
      </div>
    </div>
    <!-- <section>
      <div class="row">
        <div class="col-md-8">
          <h3 class="section-heading">Sensing organic pollutants</h3>
        </div>
      </div>
      <div class="row">
        <div class="col-md-8">
          <p>Persistent organic pollutants are harmful organic compounds of anthropogenic origins that have been, and in some cases still are, utilized as industry chemicals and pesticides. Due to their ecotoxicity and their propensity of bioaccumulation, the United Nations Environment Programme (UNEP) adopted and implemented proposed regulation of these chemicals as put forward by the Stockholm Convention on Persistent Organic Pollutants in 2001 [10]. As of 2017, 181 countries have ratified the convention [11]. While there is no explicit organic backbone, most organic pollutants share certain characteristics. They have one or more cyclic ring structures, most often of the aromatic nature, they are highly lipophilic and they are, to a varied extent, halogenated [12]. Consequently, there are different moieties that can be exploited when searching for homologues that act as natural ligands to gene regulatory elements. The 2012 UCL iGEM team successfully determined and utilized the homology between the NahR inducing ligand salicylate and the aromatic backbone found in most organic pollutants [9]. </p>
          <p>The <em>NahR</em> gene encodes a Lys-R type transcription regulator found in the naphthalene degrading operon of the NAH7 plasmid of various <em>Pseudomas</em> [13]. As with all Lys-R type regulators, NahR has a conserved N-terminal DNA-binding helix-turn-helix motif and a C-terminal co-inducer-binding domain [14]. The catabolic genes are organized in two operons, <em>fig. 1</em>, <em>nah</em> (<em>nah A-F) </em>and <em>sal</em> (<em>sal G-M</em>) that encode enzymes for metabolism of naphthalene to salicylate, and salicylate to the TCA-intermediates pyruvate and acetaldehyde respectively [13]. NahR regulates the expression of both <em>sal </em>and <em>nah</em> through binding 60 bp upstream of each respective promoter, <em>Psal </em>and<em> Pnah</em>, and is constitutively expressed upstream <em>nah-G</em> [15]<em>.</em> It binds regardless of the presence of the inducer salicylate. However, the transcriptional induction rate is approximately 20 times higher upon association with salicylate, as this induces a conformational change in the DNA-NahR complex that relieves DNA-bending at the site and subsequently allows better DNA-RNAP interaction [16]. The conformational change springs from an additional interaction between the NahR and the DNA 35 bp upstream of the RNAP binding site [15].</p>
          <p>There lies some uncertainty in the exact configuration of the DNA-NahR association. Some reports suggest that NahR binds as a monomer, while other research indicates multimerization at the binding site. Moreover, it remains unknown whether the binding is cooperative or not [15]. For further elaboration on the issue, see <a href="/Team:Lund/Model">modeling</a>.</p>
        </div>
        <div class="col-md-4">

          <figure>
            <a href="#" data-featherlight="http://2017.igem.org/wiki/images/d/d2/T--Lund--naphoperon.png">
              <img class="img-responsive center-block img-thumbnail" src="http://2017.igem.org/wiki/images/d/d2/T--Lund--naphoperon.png">
            </a>
            <figcaption>Figure 1: Schematic representation of the organization of the upper and lower degradation pathway of naphthalene found in the Nah7 plasmid of <em>Pseudomas</em>. Both pathways are under regulation of the constitutively expressed regulator NahR [17].</figcaption>
          </figure>

          <figure>
            <div id="viewer-NahR"></div>
            <figcaption>Figure 2: 3D structure of the NahR, a transcriptional regulator spanning 306 amino acids. The protein structure is organized as follows; the N-terminal consists of a highly conserved DNA-binding helix-turn-helix motif. The DNA association is furthermore dependent on a multimerization site located along the C-terminal. Through iterative mutagenesis, the ligand-binding pocket has been determined to lie close to equidistantly between the termini [18] [47] [48] .</figcaption>
          </figure>
        </div>
      </div>
    </section>
    <section>
      <div class="row">
        <div class="col-md-8">
          <h3 class="section-heading">Sensing plasticizers</h3>
        </div>
      </div>
      <div class="row">
        <div class="col-md-8">
          <p>Plasticizers are very common additives found in most commercial plastic products. They provide the polymers with durability, elasticity and flexibility, offering increased malleability and strength [19]. The plasticizer action on the polymeric material has generally thought to be lubrication between the polymeric chains, but recent thermodynamic simulations have shown that the situation might be more complex [20]. The most predominant class of plasticizers used today is that of low-weight phthalate esters (PEs), in particular DEHP (bis-2-ethylhexyl phthalate) [19]. While PEs offer a cost-effective solution to maintaining flexibility, there is cause for concern as they are not covalently bonded to the plastic and will readily leach into the environment. Multiple sources have reported severe adverse effects in vertebrates; among other things, toxic action on the human endocrine system has been noted [21] [22] [23] [24] [25]. In particular, low-weight PEs have demonstrated some antagonistic affinity for the ligand-binding domain of the human estrogen-alpha receptor (hER-α) [23] [24], making it a viable candidate for detecting plasticizers.</p>
          <p>The estrogen receptor is a member of the intracellular receptor family and exists in two different forms, hER-α and hER-β, with significant overlapping sequence homology. Each receptor shares a common architecture consisting of five domains; the N-terminal <em>A/B</em> region able to regulate gene transcription in absence of a ligand, the <em>C</em>-domain able to bind to a designated DNA sequence (DBD) and a hinge region <em>D</em> (DBD) that connects the C-domain and the ligand binding domain (LBD) <em>E</em> [26]. The mechanism of gene transactivation by the estrogen receptor has been studied in detail through analysis of the ligand-induced conformational change of the LBD. The LBD has a similar organization to that of the other nuclear receptor LBDs, with a three-layered antiparallel α-helical sandwich motif, for a total of 11 α-helices (H1-11) with one additional anti-parallel β-sheet flanking the arrangement (S1-2). The central core, located between the two outer helical layers, constitutes a wedge-shaped molecular scaffold to bind the ligand [27]. Upon interaction between the binding site and a ligand, a sizeable conformational change occurs at the C-terminal helix H12 to modify the action at the DNA-interface. An agonistic ligand spatially displaces H12 to fit snugly over the cavity whereas an antagonistic ligand prevents such confirmation and instead forces the helix to reposition through a 130° degree rotation toward the N-terminus [27][28]. Such displacement of H12 has been utilized in complementation assays as a <em>molecular switch</em>. Complementation of a split protein fused to each respective side of a truncated hER-α LBD is made possible upon antagonist association with the binding site as the termini are subsequently brought in close proximity to one another [29] [30] [31].</p>

        </div>
        <div class="col-md-8 col-md-4">
          <figure>
            <div id="viewer-hER"></div>
            <figcaption>Figure 3: 3D structure of the human estrogen receptor alpha ligand-binding domain (in complex with estradiol), a domain found in the transcriptional transactivator hER-α spanning 300 amino acids. The ligand-binding domain consists of a deep and promiscuous ligand-binding pocket consisting of a three-layered antiparallel α-helical sandwich motif. Upon recognition of a ligand, helix H12 will change conformation in accordance with the nature of the ligand; antagonists will displace the helix and bring the two termini close to one another whereas agonists will induce helix H12 to transverse the protein and stabilize over the pocket [27] [49] [50] [51] [52] .</figcaption>
          </figure>

          <figure>
            <a href="#" data-featherlight="http://2017.igem.org/wiki/images/1/1a/T--Lund--something.png">
              <img class="img-responsive center-block img-thumbnail" src="http://2017.igem.org/wiki/images/1/1a/T--Lund--something.png">
            </a>
            <figcaption>Figure 4: The hER-a ligand binding has been implemented and utilized successfully in complementation assays through capitalizing on the conformational change upon ligand recognition. [32]</figcaption>
          </figure>

        </div>
      </div>
    </section> -->

  </div>
</div>

</html>
{{Lund/footer}}
