{{Lund}}
<html>
</div> <!-- iGEM stuff-->

<script type="text/javascript">
  $('#navbar').addClass('navbar-transparent');
</script>

<header class="modeling-landing">
  <div class="container">
    <h1 class="display">Modeling</h1>
  </div>
</header>

<div class="container">
  <ul class="nav nav-pills nav-auto-stacked">
    <li role="presentation"><a href="/Team:Lund/Model">Results</a></li>
    <li role="presentation"><a href="/Team:Lund/Modeling/ModelDerivation">Model derivation</a></li>
    <li role="presentation"><a href="/Team:Lund/Modeling/ParameterEstimation">Parameter estimation</a></li>
    <li role="presentation" class="active"><a href="/Team:Lund/Modeling/Implementation">Implementation</a></li>
  </ul>
</div>

<div id="modeling" class="content-wrapper under-nav text-layout">
  <div class="container">
    <div class="row">
      <div class="col-lg-10">
        <section>
          <h3 class="section-heading">Overview</h3>

          <p>Numerical reliability and reproducibility is a crucial part of research. In this section we provide a detailed description of the implementation
          of our model.</p>
        </section>

        <section>
          <h3 class="section-heading">Programming environment</h3>

          <p> The entire system was implemented in MATLAB (2017a, The Mathworks Inc.) using the SimBiology<super>&copy;</super> app which provides a block diagram environment for a
          model-based design implementation of biochemical networks. By using drag and drop features, species and reactions are placed in an editor and
          connected to yield a model based representation of the system. The representation then generates code which constitutes to the differential
          equations (or transitions in the case of stochastic modeling) which can be used for simulations. The SimBiology app thus  provides an efficient way
          of building biochemical network models and automatically generate differential equations without the need to manually balance
          reaction fluxes which otherwise can be a highly error prone process.</p>

          <p><i>Fig. 1</i> illustrates our model implemented in the graphical user interface. The diagram consists of  blue boxes, yellow nodes and arrows
          interconnecting the components. The boxes represent the interacting species and the nodes the corresponding reactions. Lines pointing from a specie
          into a node represents a reactant and lines pointing out from a node to a specie represents a product. Dashed lines indicate that the reactant is not
          consumed under the reaction.</p>

          <p>The illustrated system can be subdivided into five parts. The first part (a) illustrates the <i>lacI</i> dynamics which expresses the <i>lac</i>
          repressor. In (b), the IPTG induction of the repressed genes represented by their operons ON and OE is shown. These operons regulate the epression
          of GFP1-9 and GFP10-ER-GFP11 respectively. The NahR pathway is marked by (c) and shows the binding to NahR to the <i>Psal</i> promoter followed by
          the subsequent transcriptional activation by salicylate to yield GFP1-9. In (d), GFP10-ER-GFP11 is translated and activates by the addition of
          estrogen. In the final part (e), the GFP association occurs, yielding a fully restored fluorophore.</p>

          <figure>
            <a href="#" data-featherlight="http://2017.igem.org/wiki/images/e/e4/T--Lund--model.svg">
              <img class="img-responsive center-block" src="http://2017.igem.org/wiki/images/e/e4/T--Lund--model.svg">
            </a>
            <figcaption> <i>Figure 1: Illustration
            of the SimBiology&copy; implementation of our system. The networks can be considered as five parts (a)-(e) governing the dynamics of separate
            components. Those include the expression of the <i>lac</i> repressor, <i>lac</i> regulation, NahR and GFP10-ER-GFP11 pathway and final GFP
            association respectively.</i> </figcaption>
          </figure>
        </section>

        <section>
          <h3 class="section-heading">Experimental setup</h3>
          <h5>Simulations</h5>
          <p>To properly emulate the working conditions of the cell at the time of addition in the wet lab, the system was run until steady state without the addition of
          inducers. The steady state values were then saved and a new simulation was started using the steady state values as initial concentrations and
          adding IPTG, salicylate and estrogen to induce gene expression. The inital simulation was run with initial concentrations set to zero expect for the
          gene content which was set at 83.2 nM for $\ce{O_N}$, $\ce{O_E}$ and $\ce{I_R}$ (see <a href="/Team:Lund/Modeling/ParameterEstimation">parameter section</a> for details).</p>

          <h5>Calculation of rise time and settling time</h5>
          <p>To measure the responsiveness of the biosensor we used the rise time and settling time as
          measures. The rise time is defined as the time it takes for a signal to go from 10% to 90% of the steady state value. The settling time is defined
          at the time it takes for a signal to fall within 2% of the steady state value. These entities were calculated using the function <i>STEPINFO</i>
          (MATLAB 2017a, The Mathworks Inc.). </p>


          <h5>Calculation of global sensitivities</h5> <p>The global sensitivities were calculated using the Sensitivity Analysis Library (SALib, see [1])
          using Python 3.5.3. The method used was the Sobol method which is a variance based sensitivity analysis method. This was chosen because it has been
          recommended to use variance based methods whenever the computational cost is not a problem [2]. The input parameters were chosen by logarithmically
          sampling the parameter space using Saltelli's sampling scheme. For more details, see the <a
          href="http://salib.readthedocs.io/en/latest/_modules/SALib/sample/saltelli.html">documentation page</a>.</p>
        </section>
        <section>
          <h3 class="section-heading">Implementation details</h3>

          <h5>Choice of ODE solver</h5>

          <p> Since we had parameters differencing with several orders of magnitude, numerical stiffness was expected and a stiff ordinary differential
          equation (ODE) solver had to be used. For this we choose MATLABs solver <i>ODE23t</i> (MATLAB 2017a, The Mathworks Inc.) as it was found to have the
          best performance on our system among the other solvers in the MATLAB ecosystem. This solver is commonly used to solve moderately stiff ODEs and
          uses a variant of the classical trapezoidal rule. </p>

          <h5>Choice of optimization algorithm</h5>

          <p>In performing the parameter estimation, several optimization algorithms were evaluated and tried such as <i>FMINSEARCH</i> (MATLAB 2017a, The
          Mathworks Inc.), <i>LSQNONLIN</i> (MATLAB 2017a, The Mathworks Inc.) and <i>PARTICLESWARM</i> (MATLAB 2017a, The Mathworks Inc.). The function
          <i>LSQNONLIN</i> was chosen as it seemed to perform best. The solver is commonly used to solve nonlinear least square problems and uses the
          Levenberg-Marquart algorithm. All optimizations were run using standard configurations. </p>

          <h5>Simulation parameters</h5>

          <p> To ensure numerically reliable solutions, all simulations were carried out using an absolute error tolerance of 10<sup>-8</sup> and carefully
          inspected to keep off spurious solutions. In addition, several solutions were tested against other ODE solvers such as <i>ODE15s</i> (MATLAB 2017a,
          The Mathworks Inc.) to ensure reliability. </p>
        </section>

        <section>
          <h3 class="section-heading">References</h3>
          <button id="refBtn" class="btn btn-default" type="button" data-toggle="collapse" data-target="#ref" aria-expanded="false" aria-controls="ref">
            Show
          </button>
          <div class="collapse" id="ref">
            <ol class="references">
              <li>[1] Herman, J. and Usher, W. (2017) SALib: An open-source Python library for sensitivity analysis. <i>Journal of Open Source Software</i>, 2(9).</li>
              <li>[2] Zi, Z. (2011). Sensitivity analysis approaches applied to systems biology models. <i>IET Systems Biology</i>, 5(6), pp.336-346. </li>
            </ol>
          </div>
        </section>
      </div>
    </div>
  </div>
</div>
</div>
<script>
  $(function() {
    $('#ref').on('shown.bs.collapse', function () {
       $("#refBtn").text('Hide');
    });

    $('#ref').on('hidden.bs.collapse', function () {
       $("#refBtn").text('Show');
    });
  });
</script>
</html>
{{Lund/footer}}
